presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/AlternativeNullableLongStateSerializer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import com.facebook.presto.common.block.Block;
+import com.facebook.presto.common.block.BlockBuilder;
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.spi.function.AccumulatorStateSerializer;
+
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+
+public class AlternativeNullableLongStateSerializer
+        implements AccumulatorStateSerializer<NullableLongState>
```

Comment:
why is the type parameter NullableLongState and not AlternativeNullableLongState?

Reply:
I tried but the compilation would fail, so I think the generating logic on `AbstractMinMaxAggregationFunction` has hard-coded name convention which led to the failing of `AlternativeNullableLongState`

So now `AlternativeNullableLongState ` is a dummy node that's only used to generate the corresponding name `AlternativeNullableLongStateSerializer `